core/zms/src/main/rdl/Domain.rdli

```diff
@@ -319,4 +319,20 @@ resource AuthHistoryDependencies GET "/domain/{domainName}/history/auth" {
         ResourceError UNAUTHORIZED;
         ResourceError TOO_MANY_REQUESTS;
     }
+}
+
+type ExpiredMembers Struct {
+    Array<MemberName> expiredRoleMembers; //list of deleted expired role members names
+    Array<GroupMemberName> expiredGroupMembers; //list of deleted expired groups members names
+}
+
+// Delete expired principals
+resource ExpiredMembers DELETE "/domain/expired-members?purgeResources={purgeResources}" {
```

Comment:
There is a conflict with the deleteTopLevelDomain endpoint, so /domain/expired-members will be changed to /expired-members

Reply:
what is the use of the purgeResources? wasn't clear from the description. Is it a boolean flag or includes multiple values? if multiple values then update the description to include what those values are